book_source/06_reference/04_docker/05_building_images.Rmd

```diff
@@ -0,0 +1,141 @@
+## Building and modifying images {#docker-build-images}
+
+For general use, it is sufficient to use the pre-built PEcAn images hosted on [Docker Hub](https://hub.docker.com/r/pecan/) (see [Docker quickstart](#docker-quickstart)).
+However, there are cases where it makes sense to re-build the Docker images locally.
```

Comment:
The easiest way to rebuild the images is to use the `docker.sh` script in the pecan folder. This script will build all the docker images (and TAG them with latest) locally on your machine. This will not build the pecan/depends image by default since it will take some time. You can trigger it to build it as well by using the following command `DEPEND=1 ./docker.sh` which will start building the pecan/depends image first.

Reply:
947ee6475